webapp/src/pages/Settings.tsx

```diff
@@ -199,7 +205,11 @@ const NumberField: React.FC<
   );
 };
 
-const Settings: React.FC = () => {
+type props = {
+  setOpen: (close: boolean) => void;
```

Comment:
Detail, but I notice `setOpen` is only always called with `false`. That makes sense since the buttons in the config can only close de config. Inspired by the similar function on `Dialog`, I would call it `onClose`. What do you think?
```suggestion
  onClose: () => void;
```